sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/StringConcatenationInLoop.cs

```diff
@@ -21,6 +21,11 @@ public StringConcatenationInLoop()
             }
             s += "a";
 
+            while (true)
+            {
+                // See https://github.com/SonarSource/sonar-csharp/issues/1138
+                s = s ?? "b";
```

Comment:
Please try to fix Vb.Net too and add a test case.

Reply:
There is no null-coalescing operator in VB.NET. The equivalent (`If`) doesn't cause a problem.